src/float.rs

```diff
@@ -2243,4 +2283,28 @@ mod tests {
         check::<f32>(1e-6);
         check::<f64>(1e-12);
     }
+
+    #[cfg(all(any(feature = "std", feature = "libm"), has_copysign))]
```

Comment:
Why does the test need `std`/`libm`? Shouldn't the default implementation work too?

Reply:
Oh, of course, that was gated because `Float` itself has to exist. Sorry for the misdirection. And the default impl will still be tested on older `std`-only CI.

Reply:
Shouldn't need to check `has_copysign` though.